consent.go

```diff
@@ -16,14 +18,14 @@ type Consent struct {
 	TcfPolicyVersion       int
 	IsServiceSpecific      bool
 	UseNonStandardStacks   bool
-	SpecialFeatureOptIns   map[int]bool
-	PurposesConsent        map[int]bool
-	PurposesLITransparency map[int]bool
+	SpecialFeatureOptIns   *Bits
```

Comment:
Is there a reason for having this a pointer type instead of just plain `Bits`?

Reply:
✔️ Done
( I always fear copying big objects 😉 )